company/views.py

```diff
@@ -485,14 +496,23 @@ def get(self, request, id, *args, **kwargs):
     @validate_company_user
     @check_company_or_manager
     def post(self, request, id, *args, **kwargs):
+        domain_id = kwargs.get("domain_id")
         domain_data = {
             "name": request.POST.get("domain_name", None),
             "url": request.POST.get("domain_url", None),
             "github": request.POST.get("github_url", None),
             "twitter": request.POST.get("twitter_url", None),
             "facebook": request.POST.get("facebook_url", None),
+            "trademarkInput": request.POST.get("trademarkInput", None),
+            "remove-button": request.POST.get("removedTrademarks", None),
         }
 
+        removed_trademarks = domain_data["remove-button"]
+        if removed_trademarks:
+            trademark_ids = removed_trademarks.split(",")
+            for trademark_id in trademark_ids:
+                Trademark.objects.filter(id=trademark_id).delete()
```

Comment:
You should be able to do the same using one query:

Trademark.objects.filter(id__in=removed_trademarks.split(",")).delete()

Reply:
Thanks.